Weather/Views/MenuHeaderView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  MenuHeaderView.swift
+//  Weather
+//
+//  Created by Oleshkevich Artem on 26.07.22.
+//
+
+import SwiftUI
+
+struct MenuHeaderView: View {
+    
+    @ObservedObject var cityViewModel: CityViewViewModel
+    
+    @State private var searchTerm = "Paris"
+    
+    func delay(delay: Double, closure: @escaping () -> ()) {
+        DispatchQueue.main.asyncAfter(deadline: .now() + delay) {
+             closure()
+        }
+    }
+    
+    var body: some View {
+        HStack {
+            Image(systemName: "magnifyingglass")
+                .foregroundColor(.white)
+                .padding()
+            
+            TextField("Paris", text: $searchTerm)
```

Comment:
Why do you have `Paris` as a placeholder here? Shoudn't it be some call to action text?